tutorials/simulators/OpenFOAM.md

```diff
@@ -360,25 +262,109 @@ much more.
 
 ### Step 4: Enhancing Performance with MPI Cluster
 
-As you have experienced, simulations can take a long, long time. To further
-reduce runtime we can change our machine configuration to a MPI cluster
-with two machines:
+Since we now want to scale our simulation into multiple machines we can no longer
+let Openfoam take care of our MPI configuration. We need to do it manually to
+let our API know how to handle MPI.
 
-```python
-mpi_cluster = inductiva.resources.MPICluster(
-                  machine_type="c3d-highcpu-360",
-                  data_disk_gb=300,
-                  num_machines=2)
-mpi_cluster.start()
+The first thing we need to do is move from running `Allrun` into setting the
+commands manually with the correct MPI configuration.
 
-# Re-run the simulation with adjusted `n_vcpus`
-task = openfoam.run(
-                  input_dir=input_dir,
-                  commands=commands,
-                  n_vcpus=360,
-                  use_hwthread=True,
-                  on=mpi_cluster)
-```
+1. **MPI Configuration**:
+
+    The first thing to do is defining the MPI configuration. This is done by:
+
+    ```python
+    config = inductiva.commands.MPIConfig("4.1.6",np=180,use_hwthread_cpus=False)
+    ```
+
+    Apart from that, we need to edit the `nCores` to 180 in the `system/include/caseDefinition`
+    file.
+
+2. **Commands**:
+
+    Now we need to define the commands to run on the cluster. This is done by
```

Comment:
Next, we need to define the commands to run on the cluster. To do this, we will need to "translate" the `Allrun` file into a list of commands to be executed on the cluster. These commands will control the simulation process.